redis_events/pyproject.toml

```diff
@@ -6,14 +6,20 @@ authors = ["Shaanjot Gill <gill.shaanjots@gmail.com>"]
 
 [tool.poetry.dependencies]
 aiohttp = "^3.7.4"
-aries-cloudagent = { version = "0.10.3" }
 fastapi = "^0.68.0"
 nest-asyncio = "^1.5.5"
 pydantic = "^1.8.2"
 python = "^3.9"
 redis = "^4.1.4"
 uvicorn = "0.16.0"
 
+# Define ACA-Py as an optional/extra dependancy so it can be
+# explicitly installed with the plugin if desired.
+aries-cloudagent = { version = ">=0.10.3, < 1.0.0", optional = true }
```

Comment:
My repo-manager script, which I think will become more important as this repo grows, works off of empty lines between sections. I don't think it would handle the comments very well either because it sorts.

But I can change the logic a bit to handle things like this better. I think this approach makes sense. We'll have to do this for the other plugins as well but can merge this for now and then do the rest if this is successful.